doc/tutorials/tutorial_X8.rst

```diff
@@ -178,6 +191,37 @@ counts:
 >>> counts[(0, 0, 0, 0, 0, 0, 0)]
 2
 
+.. note::
+
+    The :class:`~.operation` decorator allows you to create your own Strawberry Fields
+    operation. This can make it easier to ensure that the same unitary is always
+    applied to the signal and idler modes.
+
+    .. code-block:: python3
+
+        from strawberryfields.utils import operation
+
+        @operation(4)
```

Comment:
I asked this before, but I might have missed the answer. Is it not possible to make `q` an argument to the decorator rather than the qfunc?

Reply:
See discussion here: https://github.com/XanaduAI/strawberryfields/pull/330#discussion_r396824959